news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md

```diff
@@ -0,0 +1,101 @@
+     Adding block entity transmission to 1.13
+   - [#4830](https://github.com/cuberite/cuberite/pull/4830):
+     Adding Generator For Single Piece Structures
+   - [#4837](https://github.com/cuberite/cuberite/pull/4837):
+     More statistics are now tracked
+   - [#4845](https://github.com/cuberite/cuberite/pull/4845):
+     Use tracing for explosions
+   - [#4875](https://github.com/cuberite/cuberite/pull/4875):
+     Added armor and shulker box cleaning
+   - [#4881](https://github.com/cuberite/cuberite/pull/4881):
+     Adding end portal and enchanting table block entities
+   - [#4889](https://github.com/cuberite/cuberite/pull/4889):
+     Implement redstone quasi-connectivity
+   - [#4903](https://github.com/cuberite/cuberite/pull/4903):
+     Implement Dropspenser Hook
+   - [#4932](https://github.com/cuberite/cuberite/pull/4932):
+     Correct drops when using a tool with fortune
+   - [#4934](https://github.com/cuberite/cuberite/pull/4934):
+     Enchanting consumes lapis, removes correct number of levels
+   - [#4937](https://github.com/cuberite/cuberite/pull/4937):
+     Enchanting table shows detail on hover. Enchanting is deterministic.
+   - [#4946](https://github.com/cuberite/cuberite/pull/4946):
+     Adding Silverfish Spawning Blocks
+   - [#4951](https://github.com/cuberite/cuberite/pull/4951):
+     Adding wolf breading
+   - [#4973](https://github.com/cuberite/cuberite/pull/4973):
+     Added a end generator more similar to the 1.9 one
+   - [#4993](https://github.com/cuberite/cuberite/pull/4993):
+     Added the structures for the ender dragon fight
+   - [#5011](https://github.com/cuberite/cuberite/pull/5011):
+     Added new flowers on bonemeal use
+   - [#5023](https://github.com/cuberite/cuberite/pull/5023):
+     Players are no longer kicked when clicking on bedrock
+   - [#5025](https://github.com/cuberite/cuberite/pull/5025):
+     Adding Boss bar
+   - [#5055](https://github.com/cuberite/cuberite/pull/5055):
+     Added magma block contact damage
+   - [#5105](https://github.com/cuberite/cuberite/pull/5105):
+     Prepares ChunkData for BlockState storage
+   - [#5111](https://github.com/cuberite/cuberite/pull/5111):
+     Implementation of totem of undying behaviour
+   - [#5122](https://github.com/cuberite/cuberite/pull/5122):
+     Added the ability to randomize blocks in generated structures
+   - [#5124](https://github.com/cuberite/cuberite/pull/5124):
+     Basic elytra fligt
+   - [#5134](https://github.com/cuberite/cuberite/pull/5134):
+     Adds mushrooms to the generator and fixed up the roofed forest
+   - [#5179](https://github.com/cuberite/cuberite/pull/5179):
+     Teleport off of mount when throwing enderpearl
+
+ - Thanks to all the new contributors this period, as we now have 3 new 
+   entries in the CONTRIBUTORS file:
+   - 12xx12
+   - dImrich
+   - KingCol13
+
+Contribute
+----------
+
+ - If you are interested in developing Cuberite's core or plugins, 
```

Comment:
These links could be clickable, see: https://github.com/cuberite/cuberite.github.io/commit/82b6d97804b531a19057f7e925703e3a100f297a